### PR TITLE
Batch#1

### PR DESCRIPTION
**Pull request by simgobb**
https://developers.google.com/web/updates/2016/02/chrome-49-deprecations
https://developers.google.com/web/updates/2016/03/chrome-50-deprecations
https://developers.google.com/web/updates/2016/04/chrome-51-deprecations
https://developers.google.com/web/updates/2016/06/chrome-52-deprecations
https://developers.google.com/web/updates/2016/08/chrome-53-deprecations

### CODE REVIEW

**Comment by simgobb** [comment]
> @simgobb you'll need to sign the CLA as mentioned in the thread, I'll take a look at the content.

Done! Let me know if there's any other bits I need to setup. Also - I wasn't sure in terms of naming conventions etc. If there's anything you'd rather done differently, I'll just edit and get it right from the next batch

File: site/_data/authorsData.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 {
+  "paulkinlan": {
+    "homepage": "https://paul.kinlan.me/",
+    "twitter": "paul_kinlan"
+  },
+  "paulkinlan": {
+    "github": "jpmedley"
```
Comment (rachelandrew) [inline site/_data/authorsData.json:7]: this looks wrong.

File: site/en/blog/chrome-49-deprecations/index.md [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+
+In Chrome 49 (Beta Feb 2nd, 2016. Estimated stable date: March 2016) there are a
+number of changes to Chrome
+
+## Use of "css" prefix in getComputedStyle(e).cssX is deprecated
+
+**TL;DR**: The use of the "css" prefix in
+[`getComputedStyle(e)`](https://code.google.com/p/chromium/issues/detail?id=413205)
+has been deprecated since it was not a part of the formal
+[spec](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle).
```
Comment (rachelandrew) [inline site/en/blog/chrome-49-deprecations/index.md:50]:
```suggestion
[spec](https://developer.mozilla.org/docs/Web/API/Window/getComputedStyle).
```

File: site/en/blog/chrome-49-deprecations/index.md [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+[`setLocalDescription()`](https://w3c.github.io/webrtc-pc/#widl-RTCPeerConnection-setLocalDescription-Promise-void--RTCSessionDescriptionInit-description) 
+or [`setRemoteDescription()`](https://w3c.github.io/webrtc-pc/#widl-RTCPeerConnection-setRemoteDescription-Promise-void--RTCSessionDescriptionInit-description)
+without supplying an error handler. We expect to make the error handler 
+argument mandatory for these methods in Chrome 50.
+
+This is part of clearing the way for introducing promises on these methods, 
+as required by the [WebRTC spec](https://w3c.github.io/webrtc-pc/).
+
+Here's an example from the WebRTC
+[RTCPeerConnection demo](https://webrtc.github.io/samples/src/content/peerconnection/pc1/){: .external }
```
Comment (rachelandrew) [inline site/en/blog/chrome-49-deprecations/index.md:208]:
```suggestion
[RTCPeerConnection demo](https://webrtc.github.io/samples/src/content/peerconnection/pc1/)
```

File: site/en/blog/chrome-52-deprecations/index.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+## Remove non-primary button click event
+
+**TL;DR:** Non-primary mouse clicks no longer fire click events, but `MouseEvent.button` is still available.
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/pYAh8bBl5Yc/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5769439450497024) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=255)
+
+To bring Chrome in line with the [UIEvents spec](https://w3c.github.io/uievents/#event-type-click), we're removing the click events for non-primary mouse buttons. Non-primary mouse buttons vary by device. Generally this means anything other than a right or left mouse button. Note that the precise button clicked may still be retrieved using the [`MouseEvent.button` property](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button) sent to events such as [`mousedown`](https://developer.mozilla.org/en-US/docs/Web/Events/mousedown) or [`mouseup`](https://developer.mozilla.org/en-US/docs/Web/Events/mouseup).
```
Comment (rachelandrew) [inline site/en/blog/chrome-52-deprecations/index.md:104]:
```suggestion
To bring Chrome in line with the [UIEvents spec](https://w3c.github.io/uievents/#event-type-click), we're removing the click events for non-primary mouse buttons. Non-primary mouse buttons vary by device. Generally this means anything other than a right or left mouse button. Note that the precise button clicked may still be retrieved using the [`MouseEvent.button` property](https://developer.mozilla.org/docs/Web/API/MouseEvent/button) sent to events such as [`mousedown`](https://developer.mozilla.org/en-US/docs/Web/Events/mousedown) or [`mouseup`](https://developer.mozilla.org/docs/Web/Events/mouseup).
```

File: site/en/blog/chrome-53-deprecations/index.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+## FileError deprecation warning
+
+**TL;DR:** Removal of the deprecated `FileError` interface is expected in Chrome 54. Replace references to **`err`**`.code` with **`err`**`.name` and **`err`**`.message`. 
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/kJMa2rpAYqI/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/6687420359639040) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=496901)
+
+The current version of the [File API](https://w3c.github.io/FileAPI/){: .external } standard does not contain the `FileError` interface and it's support was deprecated some time in 2013. In Chrome 53, this deprecation warning will be printed to the DevTools console:
```
Comment (rachelandrew) [inline site/en/blog/chrome-53-deprecations/index.md:67]:
```suggestion
The current version of the [File API](https://w3c.github.io/FileAPI/) standard does not contain the `FileError` interface and it's support was deprecated some time in 2013. In Chrome 53, this deprecation warning will be printed to the DevTools console:
```

File: site/en/blog/chrome-49-deprecations/index.md [hunk 6]
```diff
@@ -0,0 +1,348 @@
...
+          changedTouches, 0, 0);
+      }
+    }
+    
+    document.body.dispatchEvent(touchEvent);
+ ```   
+
+## Error and success handlers required in RTCPeerConnection methods
+
+**TL;DR:** The [WebRTC](http://www.html5rocks.com/en/tutorials/webrtc/basics/){: .external }
```
Comment (rachelandrew) [inline site/en/blog/chrome-49-deprecations/index.md:191]:
```suggestion
**TL;DR:** The [WebRTC](http://www.html5rocks.com/en/tutorials/webrtc/basics/)
```

File: site/en/blog/chrome-50-deprecations/index.md [hunk 7]
```diff
@@ -0,0 +1,255 @@
...
+[Chromium Bug](https://code.google.com/p/chromium/issues/detail?id=568261)
+
+This attribute is simply an alias to the `Keyboard.prototype.location`
+attribute, which allows disambiguation between keys that are located multiple
+places on a keyboard. For example, both attributes allow developers to
+distinguish between the two `Enter` keys on an extended keyboard.
+
+## Error and success handlers required in RTCPeerConnection methods
+
+**TL;DR**: The [WebRTC](http://www.html5rocks.com/en/tutorials/webrtc/basics/){: .external }
```
Comment (rachelandrew) [inline site/en/blog/chrome-50-deprecations/index.md:150]:
```suggestion
**TL;DR**: The [WebRTC](http://www.html5rocks.com/en/tutorials/webrtc/basics/)
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
I've left a couple of comments and suggestions in there but it generally is looking good!

File: site/_data/authorsData.json (hunk 1 above)
Comment (simgobb) [reply site/_data/authorsData.json:7]: Sorry! Not sure what went wrong there.

File: site/_data/i18n/authors.yaml [hunk 8]
```diff
@@ -1,3 +1,13 @@
+josephmedley:
+  title:
+    en: 'Joseph Medley'
```
Comment (rachelandrew) [inline site/_data/i18n/authors.yaml:3]:
```suggestion
    en: 'Joe Medley'
```

File: site/_data/i18n/authors.yaml [hunk 9]
```diff
@@ -1,3 +1,13 @@
+josephmedley:
```
Comment (rachelandrew) [inline site/_data/i18n/authors.yaml:1]:
```suggestion
joemedley:
```

File: site/en/blog/chrome-50-deprecations/index.md [hunk 10]
```diff
@@ -0,0 +1,255 @@
...
+# Required
+description: >
+  An round up of the deprecations and API removals in Chrome to help you plan.
+
+# Optional
+# How to add a new author
+# https://developer.chrome.com/docs/handbook/how-to/add-an-author/
+authors:
+  - paulkinlan
+  - josephmedley
```
Comment (rachelandrew) [inline site/en/blog/chrome-50-deprecations/index.md:17]:
```suggestion
  - joemedley
```

File: site/en/blog/chrome-51-deprecations/index.md [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+
+# Required
+description: >
+  An round up of the deprecations and removals in Chrome to help you plan.
+
+# Optional
+# How to add a new author
+# https://developer.chrome.com/docs/handbook/how-to/add-an-author/
+authors:
+  - josephmedley
```
Comment (rachelandrew) [inline site/en/blog/chrome-51-deprecations/index.md:16]:
```suggestion
  - joemedley
```

File: site/en/blog/chrome-52-deprecations/index.md [hunk 12]
```diff
@@ -0,0 +1,112 @@
...
+
+# Required
+description: >
+  An round up of the deprecations and removals in Chrome to help you plan.
+
+# Optional
+# How to add a new author
+# https://developer.chrome.com/docs/handbook/how-to/add-an-author/
+authors:
+  - josephmedley
```
Comment (rachelandrew) [inline site/en/blog/chrome-52-deprecations/index.md:16]:
```suggestion
  - joemedley
```

File: site/en/blog/chrome-53-deprecations/index.md [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+
+# Required
+description: >
+  A round up of the deprecations and removals in Chrome to help you plan.
+
+# Optional
+# How to add a new author
+# https://developer.chrome.com/docs/handbook/how-to/add-an-author/
+authors:
+  - josephmedley
```
Comment (rachelandrew) [inline site/en/blog/chrome-53-deprecations/index.md:16]:
```suggestion
  - joemedley
```

**Review by rachelandrew — APPROVED** [review]
